pkg/repositories/transformers/workflow.go

```diff
@@ -47,11 +48,26 @@ func FromWorkflowModel(workflowModel models.Workflow) (admin.Workflow, error) {
 		return admin.Workflow{}, errors.NewFlyteAdminErrorf(codes.Internal, "failed to read created at timestamp")
 	}
 
+	var workflowInterface core.TypedInterface
+	if workflowModel.TypedInterface != nil && len(workflowModel.TypedInterface) > 0 {
+		err = proto.Unmarshal(workflowModel.TypedInterface, &workflowInterface)
+		if err != nil {
+			return admin.Workflow{}, errors.NewFlyteAdminErrorf(codes.Internal, fmt.Sprintf("failed to unmarshal workflow %v inputs", workflowModel.ID))
```

Comment:
```suggestion
			return admin.Workflow{}, errors.NewFlyteAdminErrorf(codes.Internal, fmt.Sprintf("failed to unmarshal workflow %v interface", workflowModel.ID))
```
since this includes inputs and outputs